_data/tools.json

```
@@ -157,7 +157,22 @@
             "AUTOMATED",
             "MANUAL"
         ],
-        "datasets": []
+        "datasets": [
+            {
+                "name": "idr0062",
+                "link": "http://idr.openmicroscopy.org/webclient/?show=project-801",
```

```suggestion
                "link": "https://idr.openmicroscopy.org/search/?query=Name:idr0062",
```

Should we start using the top-level accession URLs rather than the internal webclient ones?